### PR TITLE
chore: improve logs

### PR DESCRIPTION
**Pull request by bap2pecs**
per discussion w @SebastianElvis at https://github.com/babylonchain/finality-provider/pull/451, we should also add those changes in `dev`

### CODE REVIEW

File: itest/babylon_node_handler.go [hunk 1]
```diff
@@ -149,6 +149,7 @@ func NewBabylonNodeHandler(t *testing.T, covenantQuorum int, covenantPks []*type
 
 	f, err := os.Create(filepath.Join(testDir, "babylon.log"))
 	require.NoError(t, err)
+	t.Logf("babylon log file: %s", f.Name())
```
Comment (gitferry) [inline itest/babylon_node_handler.go:152]: leftover?

**Review by gitferry — APPROVED** [review]

File: itest/babylon_node_handler.go (hunk 1 above)
Comment (bap2pecs) [reply itest/babylon_node_handler.go:152]: no this is intentionally to print it out as I think this log is helpful for debugging

**Review by maurolacy — APPROVED** [review]
LGTM. Thanks for doing this btw.

If this is merged to `dev`, a rebase / merge will be needed to bring it to `base`.

**Comment by bap2pecs** [comment]
@maurolacy actually this is a follow-up PR for #451 which does the same thing so we can save the rebase/merge

**Comment by maurolacy** [comment]
Let's close this one without merging then.

**Comment by maurolacy** [comment]
I'm lost, as this is against dev and the other isn't. Anyway, you figure it out and merge appropiately.

**Comment by bap2pecs** [comment]
@maurolacy yes you can see my discussion with Runchao there. I created this separate PR simply by copying a few changes from that one and apply here.

image

it has to be a separate PR b/c that one has changes related to OP consumer which doesn't exist in `dev`

**Comment by maurolacy** [comment]
Oh, you meant adding the same stuff (improved logging) to both branches. Not ideal (may clash on rebase), but OK for small features I guess.

**Comment by bap2pecs** [comment]
yeah not sure if there is better way for that given that the two branches now diverges a lot

> Not ideal (may clash on rebase), but OK for small features I guess.

wait I thought copying the same code in both branches would save some headaches when we consolidate two branches. or is that not the case?

**Comment by maurolacy** [comment]
> wait I thought copying the same code in both branches would save some headaches when we consolidate two branches. or is that not the case?

Not necessarily. It can lead to conflicts. Anyway, OK for small features / changes.

**Comment by bap2pecs** [comment]
> Not necessarily. It can lead to conflicts.

isn't that similar to `git cherry-pick` a commit? I think this can help reduce merge conflicts when we eventually consolidate the two branches

**Pull request merged**